exchange/bitswap/wantmanager.go

```diff
@@ -153,7 +153,7 @@ func (mq *msgQueue) doWork(ctx context.Context) {
 
 	err := mq.network.ConnectTo(conctx, mq.p)
 	if err != nil {
-		log.Noticef("cant connect to peer %s: %s", mq.p, err)
+		log.Warningf("cant connect to peer %s: %s", mq.p, err)
```

Comment:
this should probably just be `log.Infof` -- warnings are things the user may need to be conscious about